### ISSUE TITLE
Persistent processing fails when PC ends turn

### ISSUE THREAD

**Issue by JamzTheMan** [CarlosFdez/pf2e-persistent-damage#21]
Persistent damage/healing cards to not show up/process when a Player ends turn. They only work if the GM ends the combat turn.

Tested this with no other modules active with GM & Player logged in.

*this module + find the culprit*
![image](https://user-images.githubusercontent.com/8051654/148473910-1c3d4cf6-e1f1-4c68-9560-b33fcc6ffa81.png)

**Comment by CarlosFdez** [comment]
Might have been due to some sillyness shark did with the combat tracker. I'll dig into it soon, thanks for reporting.

That said healing doesn't work right now.

**Comment by JamzTheMan** [comment]
Can you elaborate on "healing doesn't work"?  I do get the messages as long as the GM ends the turn.

FWIW I just added some chatmessage hooks trapping for your fast healing/regen messages and auto-apply the healing in my own personal module...

**Comment by CarlosFdez** [comment]
Can't reproduce.  I had a player end their turn and the persistent damage rolled just fine.

**Comment by JamzTheMan** [comment]
Really strange. I'll try it again today in a new world again.

**Comment by JamzTheMan** [comment]
OK, so it seems to only affect Fast healing/Regeneration. It happens when a Player ends turn and the next token has Fast healing/Regeneration.  I suspect this is because it happens on the next tokens start-turn and don't have permission or something. If the GM end's the players turn fast healing kicks off for next token.

I tested this in a fresh world, all new installs (module/system), only this module active, GM NPC + player 2 & player 3 PC Tokens.
I also gave player 2 ownership of player 3 token with player 3 token having fast healing and that also didn't work (where player 2 goes then player 3 then NPC, so PC having ownership doesn't seem to matter).

FYI Persistent Damage *does* seem to work as normal as that happens on your own token anyway.
